admin_guide/topics/creating-primed-images-cluster-auto-scaler.adoc

```diff
@@ -19,7 +19,7 @@ If you already have a primed image, you can use it instead of creating a new one
 On the host that you used to create your {product-title} cluster, create a
 primed image:
 
-. Create a new Ansible inventory file on your local host:
+. Create a new Ansible inventory file on your local host. This file requires variables that assign the `cloudprovider` flag to the participating nodes:
```

Comment:
I think that it would be useful to the reader to highlight why this is beneficial. I.e. what does the cloudprovider flag enable? Why is it needed?

Reply:
Thank you, Paul. This feedback has been incorporated.